statsmodels/tsa/tsatools.py

```diff
@@ -652,6 +652,10 @@ def freq_to_period(freq):
         return 12
     elif freq == 'B' or freq == 'W' or freq.startswith('W-'):
         return 52
+    elif freq == 'D' or freq == 'C' or freq.startswith('D-'):
+        return 365
```

Comment:
Is it "obvious" that daily frequency should have annual seasonality, and not weekly?
(I have no preference right now since I don't know what the usual defaults are.)